README.md

```diff
@@ -1,66 +1,83 @@
-### Running the Project
+  ```bash
+  $ npm install
+  ```
 
-To launch the application, you can use `node .` in the project's root directory. You might also consider using [nodemon](https://github.com/remy/nodemon) for this. It works just like the node command, but automatically restarts your application when you change any source code files.
+  This should install all of our project dependencies from npm into a local `node_modules` folder.
 
-```bash
-npm install -g nodemon
-nodemon .
-```
+1. To launch the application, you can use `node .` in the project's root directory. You might also consider using [nodemon](https://github.com/remy/nodemon) for this. It works just like the node command, but automatically restarts your application when you change any source code files.
+
+  ```bash
+  $ npm install -g nodemon
+  $ nodemon .
+  ```
 
 ### Docker local development
 
-First, install [Docker](https://www.docker.com/) and [docker-compose](https://docs.docker.com/compose/install/).
+1. Install [Docker](https://www.docker.com/) and [docker-compose](https://docs.docker.com/compose/install/).
+
+1. Make sure you performed step `2` on the previous section.
 
-Then set the following environment variables in your terminal session. You can find these values at https://www.twilio.com/user/account/voice and at https://dashboard.authy.com.
+1. Finally, run the following commands to start your Docker containers:
 
-- `TWILIO_NUMBER`
-- `TWILIO_ACCOUNT_SID`
-- `TWILIO_AUTH_TOKEN`
-- `AGENT_NUMBER`
+  ```
+  $ docker-compose up -d
+  ```
 
-Finally, run the following commands to start your Docker containers:
+  Warning: If you previously ran  ```npm install``` locally, the node_modules folder will conflict with the file structure of the container when you run the above command.
+  We recommended installing your node dependencies one folder up from the rest of your source code.
 
-```
-$ docker-compose up -d
-```
+1. You can then visit the application at [http://localhost:3000/](http://localhost:3000/). If you're using [boot2docker](https://docs.docker.com/installation/mac/) to run Docker on OS X, you'll need to use the value of `boot2docker ip` instead of `localhost`.
 
-Warning: If you previously ran  ```npm install``` locally, the node_modules folder will conflict with the file structure of the container when you run the above command.
-We recommended installing your node dependencies one folder up from the rest of your source code.
+1. To stop your containers, run `docker-compose stop`.
 
-You can then visit the application at [http://localhost:3000/](http://localhost:3000/). If you're using [boot2docker](https://docs.docker.com/installation/mac/) to run Docker on OS X, you'll need to use the value of `boot2docker ip` instead of `localhost`.
+## Deploy On Heroku
 
-To stop your containers, run `docker-compose stop`.
+[![Deploy](https://www.herokucdn.com/deploy/button.png)](https://heroku.com/deploy?template=https://github.com/TwilioDevEd/lead-alerts-node)
```

Comment:
Probably this is the right time to use the **.svn** version of the Heroku button.